apps/backoffice-v2/src/lib/blocks/variants/DefaultBlocks/hooks/useDefaultBlocksLogic/useDefaultBlocksLogic.tsx

```diff
@@ -297,6 +301,33 @@ export const useDefaultBlocksLogic = () => {
 
   const companySanctionsBlock = useCompanySanctionsBlock(companySanctions);
 
+  const childWorkflowToUboAdapter = (childWorkflow: TWorkflowById) => {
+    return {
+      name: [
+        childWorkflow?.context?.entity?.data?.firstName,
+        childWorkflow?.context?.entity?.data?.lastName,
+      ]
+        .filter(Boolean)
+        .join(' '),
+      nationality: childWorkflow?.context?.entity?.data?.additionalInfo?.nationality,
+      email: childWorkflow?.context?.entity?.data?.email,
+      identityNumber: childWorkflow?.context?.entity?.data?.nationalId,
+      percentageOfOwnership:
+        childWorkflow?.context?.entity?.data?.percentageOfOwnership ??
+        childWorkflow?.context?.entity?.data?.ownershipPercentage ??
+        childWorkflow?.context?.entity?.data?.additionalInfo?.percentageOfOwnership ??
+        childWorkflow?.context?.entity?.data?.additionalInfo?.ownershipPercentage,
```

Comment:
Temporary solution until fields are normalized across customers and definitions.